examples/layers/rendering/src/sector_layout.dart

```diff
@@ -585,3 +586,48 @@ class RenderSolidColor extends RenderDecoratedSector {
     }
   }
 }
+
+/// The result of performing a hit test on [RenderSector]s.
```

Comment:
It is correct that SectorHitTestResult currently does not provide any additional functionality over HitTestResult because RenderSector is an incomplete example implementation of an alternative to RenderBox. Once we have something like RenderSectorTransform (which, changes the origin of its child to be different from the parent) we'd add methods here to implement the origin changing for hit testing. I've added added a TODO for this to the class.